### PR TITLE
Make storage directory configurable

### PR DESCRIPTION
**Pull request by skeggse**
Fixes https://github.com/mixmaxhq/custody-probe/issues/2.

### CODE REVIEW

File: src/utils/storage.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const { readFile, writeFile } = promisify.methods(require('fs'), ['readFile', 'w
 const mkdirp = promisify(require('mkdirp'));
 
 // POSIX-only, but no one's using this on Windows yet.
-const STORAGE_ROOT_DIR = '/usr/local/var/custody';
+const STORAGE_ROOT_DIR = process.env.CUSTODY_PROC_DIR || '/usr/local/var/custody';
```
Comment (wearhere) [inline src/utils/storage.js]: Curious is it a requirement that Linux users always set this or only if they use custody-probe?

Comment (skeggse) [reply src/utils/storage.js]: Depending on distro, Linux users may need to set this. Currently, if the `/usr/local/var` dir doesn't exist (which is usually the case), the process will just crash.

Comment (wearhere) [reply src/utils/storage.js]: Ah ok. I was wondering if we should throw our own error if this is not set on Linux, to help users identify they needed to set it, but I guess we can't make that a general rule.

Comment (skeggse) [reply src/utils/storage.js]: I'll document here too - that should help a little.

**Pull request merged**